libs/fixed_point/src/ufp32.sw

```diff
@@ -98,7 +98,7 @@ impl core::ops::Multiply for UFP32 {
         }
 
         Self {
-            value: res_u64,
+            value: asm(ptr: res_u64) { ptr: u32 },
```

Comment:
Why assembly?

Reply:
Remove warnings of converting `u64` to `u32`